app/controllers/api/v0/github_repos_controller.rb

```diff
@@ -3,10 +3,12 @@ module V0
     class GithubReposController < ApiController
       include Pundit
 
+      before_action :authenticate_with_api_key_or_current_user!
```

Comment:
Let's use a before_action to require `current_user` for this controller so its not opened up to API key holders and we still get the fix we want.